base/sparse/linalg.jl

```diff
@@ -941,3 +941,17 @@ end
 chol(A::SparseMatrixCSC) = error("Use cholfact() instead of chol() for sparse matrices.")
 lu(A::SparseMatrixCSC) = error("Use lufact() instead of lu() for sparse matrices.")
 eig(A::SparseMatrixCSC) = error("Use eigs() instead of eig() for sparse matrices.")
+
+function Base.cov(X::SparseMatrixCSC, vardim::Int=1; corrected::Bool=true)
+    a, b = size(X)
+    n, p = vardim == 1 ? (a, b) : (b, a)
+    out = Matrix(Base.unscaled_covzm(X,vardim)) # part1
```

Comment:
Space after comma. Also `part1` cannot be understood by reading a code. Maybe add a comment before the function showing the formula with its three parts?